# 4. Написать функцию которая принимает на вход число от 1 до 100.
# Если число равно 13, функция поднимает исключительную ситуации ValueError
# иначе возвращает введенное число, возведенное в квадрат.
# Далее написать основной код программы. Пользователь вводит число.
# Введенное число передаем параметром в написанную функцию и печатаем результат,
# который вернула функция. Обработать возможность возникновения исключительной ситуации,
# которая поднимается внутри функции.

def pow_number(x):
    if x == 13:
        raise ValueError('Несчастливое число!')
    else:
        return pow(x,2)

number = int(input('Введите число от 1 до 100: '))

try:
    result = pow_number(number)
except ValueError:
    print('У нас несчастливое число!')
else:
    print(result)

